exercise/static/exercise/chapter.js

```diff
@@ -158,6 +137,9 @@
 		content_element: '<div class="exercise-content"></div>',
 		content_selector: '.exercise-content',
 		exercise_selector: '#exercise-all',
+		quiz_progress_selector: '#quiz-submit-progress',
```

Comment:
I think, thise should be classes as there can be multiple of these on a single page.

Reply:
I changed these now